packages/sourcecred/src/api/main/graph.js

```diff
@@ -50,7 +50,7 @@ export async function graph(
   // Build graphs
   const pluginGraphs = [];
   for (const {plugin, directoryContext} of input.plugins) {
-    if (scope.includes(plugin.id)) {
+    if (!scope || scope.includes(plugin.id)) {
```

Comment:
when I see that scope is an optional array, I intuitively expect that if scope is unset, it will default to an empty value, i.e. an empty array. however, the semantics are that if scope is unset, then every plugin is implicitly in scope, which is counter-intuitive. can you please add a docstring explaining the semantics here, and what happens if scope is unset?

also, from the PR description it's not clear why this change was needed.